### DISCUSSION TITLE
API Stability

### DISCUSSION THREAD

**Discussion by johnwason** [tesseract-robotics/tesseract#1068] [General]
I started to look at updating the Python wrappers and found way more API changes than I expected. A project that has been around for this long should not have such an unstable API. It makes it impossible for users to maintain any code that depends on the project, and will discourage people from depending on it.

**Comment by Levi-Armstrong** [comment]
I believe the project is close to 1.0 release but there are still some things that will need to be changed from an API perspective mostly in the tesseract_planning repository before this can happen. So to help with this, anytime the API is changed we have been incrementing the minor version so those that depend on Tesseract can fix their projects to a tag until they are ready to upgrade.
